src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -475,7 +471,7 @@ where
             .narrow(&Step::ModulusConvertBreakdownKeyBitsAndTriggerValues)
             .set_total_records(num_outputs),
         flattenned_stream,
-        0..BK::BITS + TV::BITS,
+        0..<BK as WeakSharedValue>::BITS + <TV as WeakSharedValue>::BITS,
```

Comment:
```suggestion
        0..(<BK as WeakSharedValue>::BITS + <TV as WeakSharedValue>::BITS),
```

Too complex to track now.